src/Components/NewCustomerForm.js

```diff
@@ -0,0 +1,153 @@
+import React, { useState } from "react";
+const NewCustomerForm = ({ handleNewCustomerSubmit }) => {
+  const [newCustomer, setNewCustomer] = useState({
+    id: 0,
+    title: "",
+    firstName: "",
+    surname: "",
+    email: "",
+    roomId: 0,
+    checkInDate: "",
+    checkOutDate: ""
+  });
+
+  const handleChange = event => {
+    const updatedNewCustomer = {
+      ...newCustomer,
+      [event.target.name]: event.target.value
+    };
+    setNewCustomer(updatedNewCustomer);
+  };
+
+  return (
+    <div className="container mt-5 mb-5">
+      <h3>Add a new customer below:</h3>
+      <form
+        id="newCustomer"
+        onSubmit={e => {
+          e.preventDefault();
+          handleNewCustomerSubmit(newCustomer);
```

Comment:
Well done on creating the form and adding a new booking to the list!👏 It wasn't an easy task but you've managed to figure it out🏅 The only thing I would say is that you could set all values back to empty strings when the form is submitted as it doesn't clear the fields at the moment